Monika After Story/game/screens.rpy

```diff
@@ -1484,6 +1486,67 @@ screen notif_settings():
         xoffset 300 yoffset -10
         style "main_menu_version"
 
+## hotkeys helper screen
+screen hot_keys():
+    tag menu
+
+    use game_menu(("Hotkeys"), scroll="viewport"):
+
+        default tooltip = Tooltip("")
+
+        # making each indivual list a vbox essentially lets us auto-align
+        vbox:
+            spacing 25
+
+            hbox:
+                style_prefix mas_ui.cbx_style_prefix
+                vbox:
+                    label _("General")
+                    spacing 10
+                    text _("Music")
+                    text _("Play")
+                    text _("Talk")
+                    text _("Bookmark")
+                    text _("Derandom")
+                    text _("Fullscreen")
+                    text _("Screenshot")
+                    text _("Settings")
+
+                vbox:
+                    label _("")
+                    spacing 10
+                    text _("M")
+                    text _("P")
+                    text _("T")
+                    text _("B")
+                    text _("X")
+                    text _("F")
+                    text _("S")
+                    text _("Esc")
+
+            hbox:
+                style_prefix mas_ui.cbx_style_prefix
+                vbox:
+                    label _("Music")
+                    spacing 10
+                    text _("Vol Up")
+                    text _("Vol Down")
```

Comment:
can you spell out "Volume" here for clarity? We have enough space to use the full word anyway.

Reply:
Ah yeah, I did that originally when I made the menu a different way, but should fit now.